src/routes/profile/+page.svelte

```diff
@@ -99,89 +117,124 @@
 	}
 	async function loadMorePosts() {
 		postData = await loadPosts(get(token), postData, 'mabu2807');
-		console.log('MaxPostCounter: ' + maxPostCounter);
 		maxPostCounter = Number(maxPostCounter) + Number(postData.pagination.limit);
-		console.log('MaxPostCounter: ' + maxPostCounter);
-		console.log('Postdata: ' + postData.records.length);
+	}
+	async function subscribe() {
+		const followStatus = await followUser(get(token), profileData.user.subscriptionId);
+		if (followStatus == 200) {
+			subcireped = true;
+			toastStore.trigger(createToast('User was followed', 'success'));
```

Comment:
translations einfügen

Reply:
gilt auch für alle weiteren

Reply:
Leider nicht so einfach, bei toasts übersetzungen hinzuzufügen deswegen erstmal nur englisch, aber ich hab sie in der translation.ts schonmal angelegt